itests/wdpost_test.go

```diff
@@ -63,7 +65,7 @@ func testWindowPostUpgrade(t *testing.T, blocktime time.Duration, nSectors int,
 	require.NoError(t, err)
 
 	t.Log("Running one proving period")
-	waitUntil := di.PeriodStart + di.WPoStProvingPeriod + 2
```

Comment:
Was this what fixed the "proving deadline 0 in deadline 1" issue?

Reply:
You can see that issue in the "+20". Termination happens at the end of the deadline _always_. If we need "slack", something is wrong.